operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -152,6 +180,16 @@ querier:
   query_ingesters_within: 3h
   tail_max_duration: 1h
   max_concurrent: {{ .MaxConcurrent.AvailableQuerierCPUCores }}
+{{- if .Gates.GRPCEncryption }}
+compactor_client:
+  tls_enabled: true
+  tls_cert_path: {{ .TLS.Paths.GRPC.Certificate }}
+  tls_key_path: {{ .TLS.Paths.GRPC.Key }}
+  tls_ca_path: {{ .TLS.Paths.CA }}
+  tls_server_name: {{ .TLS.ServerNames.HTTP.Compactor }}
+  tls_cipher_suites: {{ .TLS.CipherSuitesString }}
+  tls_min_version: {{ .TLS.MinTLSVersion }}
+{{- end }}
```

Comment:
The compactor client is unfortunately still a HTTP client although it should be a GRPC one (See #7595). Therefore this one should be gated by `HTTPEncryption` and using the `TLS.Paths.HTTP` fields instead.